packages/desktop-client/src/components/spreadsheet/useSheetValue.ts

```diff
@@ -7,7 +7,10 @@ import { useSheetName } from './useSheetName';
 
 import { type Binding } from '.';
 
-export function useSheetValue(binding: Binding, onChange?: (result) => void) {
+export function useSheetValue<T>(
+  binding: Binding,
+  onChange?: (result) => void,
+): T | null {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏any way we could make this _smarter_? For example: I see that `binding` can have a `value`. In such case we should be able to extract the type from there. Perhaps we can also extract the type from the binding itself my the name of it?

Basically I'm afraid that manual type definitions here will lead to subtle bugs later.